gui-js/libs/ui-components/src/lib/ui-components.module.ts

```diff
@@ -12,7 +14,6 @@ import { EditUserFunctionComponent } from './edit-user-function/edit-user-functi
 import { EquationsComponent } from './equations/equations.component';
 import { FindAllInstancesComponent } from './find-all-instances/find-all-instances.component';
 import { GodleyWidgetViewComponent } from './godley-widget-view/godley-widget-view.component';
-import { VariablePaneComponent } from './variable-pane/variable-pane.component';
```

Comment:
I suppose variable-pane.component being included into ui-components/src/index.ts suffices to include this module into the software. What is the purpose of this file then?

Reply:
I think the only thing the presence of the file in intex.ts enables is the possibility to import the component with an alias... including the components in a module happens in the .module.ts files like this one. Specifically the component needs to be in declarations to be used within the module itself, and in exports to be used in other modules that reference the current one.
Generally the import alias is not used (by convention) for files within the same module... within a module we tend to use relative paths, although aliasing is allowed...

Reply:
example of using the alias:
import { events, MinskyProcessPayload, DescriptionPayload, HandleDescriptionPayload, HandleDimensionPayload, PickSlicesPayload, LockHandlesPayload } from '@minsky/shared';